src/UI/Art/SpriteArtist/CommanderOverlayArtist.java

```diff
@@ -173,18 +172,18 @@ private static BufferedImage getActiveAbilityName(String abilityName)
       Graphics fgGfx = foreground.getGraphics();
 
       // Generate a shaped black background to hold the ability name.
-      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 0, 0, 1);
-      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 0, 1, 1);
-      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 0, 2, 1);
-      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 1, 0, 1);
-      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 1, 1, 1);
-      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 1, 2, 1);
-      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 2, 0, 1);
-      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 2, 1, 1);
-      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 2, 2, 1);
+      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 0, 0);
+      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 0, 1);
+      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 0, 2);
+      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 1, 0);
+      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 1, 1);
+      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 1, 2);
+      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 2, 0);
+      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 2, 1);
+      SpriteLibrary.drawTextSmallCaps(bgGfx, abilityName, 2, 2);
```

Comment:
...oh man, I definitely would have complained about this before, if I'd been paying enough attention.
I did vaguely wonder where the ability text font came from. Now I know.

The unit-marking font is easily recolorable, maybe swap to that?
Then again, this ain't broken, so maybe I'm making mountains out of molehills here.

Reply:
The unit-marking font is also much wider; not sure it would look as nice there.
Out of scope for this PR, at any rate.

Reply:
Both good points.